fr.free.hxc2001.HxCFloppyEmulator.Extension.capslib.yaml

```
@@ -0,0 +1,35 @@
+id: fr.free.hxc2001.HxCFloppyEmulator.Extension.capslib
+branch: 1-19.08
+runtime: fr.free.hxc2001.HxCFloppyEmulator
+runtime-version: stable
+sdk: org.freedesktop.Sdk//19.08
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: ${FLATPAK_DEST}
+modules:
+  - name: capslib
+    buildsystem: simple
+    build-commands: 
+      - install -D apply_extra "${FLATPAK_DEST}/bin/apply_extra"
+    post-install:
+      - install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo fr.free.hxc2001.HxCFloppyEmulator.Extension.capslib.metainfo.xml
+      - appstream-compose --basename=fr.free.hxc2001.HxCFloppyEmulator.Extension.capslib --prefix=${FLATPAK_DEST} --origin=flatpak fr.free.hxc2001.HxCFloppyEmulator.Extension.capslib
+    sources: 
+      - type: extra-data
+        only-arches: [x86_64]
+        url: http://www.softpres.org/_media/files:ipflib42_linux-x86_64.tar.gz
+        sha256: ea139673e29a36dc3be7ba916628b5c8a03353bd68be4d346daed803d472ddf3
+        size: 48272
+        filename: ipflib42_linux.tar.gz
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - tar -xzf ipflib42_linux.tar.gz
+          - install -d lib
+          - install -Dm644 -t lib x86_64-linux-gnu-capsimage/libcapsimage.so.4.2
+          - ln -s lib/libcapsimage.so.4.2 lib/libcapsimage.so.5.1
```

Is this correct? I believe it should be
```suggestion
          - ln -s libcapsimage.so.4.2 lib/libcapsimage.so.5.1
```

---

Yeah, something I'm doing incorrectly. 
* In the app's script I have `add-ld-path: extra/lib`
* In the extension manifest I create the dir `lib` in then install the library to it
* Then I create a symbolic link to it since HxC looks for `5.1` when that is not readily available, so I have to use `4.2`

---

You currently create broken `lib/libcapsimage.so.5.1` symlink, it points to `lib/lib/libcapsimage.so.4.2`. Either use `ln -sr` or apply my suggestion.

---

Oh yes, of course. That took a long time for my brain to get :sweat_smile: